### PR TITLE
Release GCS connector 2.2.5

### PR DESCRIPTION
**Pull request by Deependra-Patel**

### CODE REVIEW

File: gcs/CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-### 2.2.5 - 2021-XX-XX
+### 2.2.6 - 2022-XX-XX
+
+### 2.2.5 - 2022-01-31
 
 1.  Enable TCP keep alive by default.
```
Comment (mprashanthsagar) [inline gcs/CHANGES.md:5]: Do we want to add any other updates to changes.md ?

Comment (Deependra-Patel) [reply gcs/CHANGES.md:5]: We can maybe Add message level timeouts and support for traffic director?
https://github.com/GoogleCloudDataproc/hadoop-connectors/commits/branch-2.2.x/gcs

**Pull request merged**